modules/afmongodb/afmongodb.c

```diff
@@ -611,9 +711,9 @@ afmongodb_dd_init(LogPipe *s)
           for (l=self->servers; l; l = g_list_next(l))
             {
               gchar *host = NULL;
-              gint port = 27017;
+              gint port = MONGOC_DEFAULT_PORT;
 
```

Comment:
Based on this, `MONGOC_DEFAULT_PORT` cannot be used in place of `MONGO_CONN_LOCAL`.
MongoDB provides a way for local connections via a unix domain socket. When  `MONGO_CONN_LOCAL` was set as port, then we connect to mongodb via unix domain socket (value for `MONGO_CONN_LOCAL` is set for -1 in the libmongo client) .